### PR TITLE
2023_09 budget update proposal

### PR DESCRIPTION
**Pull request by Valdorff**

### CODE REVIEW

File: assets/rpip-10/expense_2.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+## Rationale
+There was a mismatch between the actual budget (in RPIP) and the intent (in vote text) as explained
+in <https://dao.rocketpool.net/t/intended-vs-actual-budget-mismatch-in-rpip-10/2053?u=valdorff>.
+
+This expense compensates the difference during periods affected before the budget was updated.
+
+| Period | RPL to treasury | IMC amount to make up | GMC amount to make up |
+|--------|-----------------|-----------------------|-----------------------|
+| n      | x               | `x*(.34-.27)=.07x`    | `x*(.205-.16)=.045x`  |
```
Comment (xrchz) [inline assets/rpip-10/expense_2.md:13]: Where do these formulas come from? I can't see them easily in the linked explanation.

File: assets/rpip-10/expense_2.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+## Rationale
+There was a mismatch between the actual budget (in RPIP) and the intent (in vote text) as explained
+in <https://dao.rocketpool.net/t/intended-vs-actual-budget-mismatch-in-rpip-10/2053?u=valdorff>.
+
+This expense compensates the difference during periods affected before the budget was updated.
+
+| Period | RPL to treasury | IMC amount to make up | GMC amount to make up |
+|--------|-----------------|-----------------------|-----------------------|
+| n      | x               | `x*(.34-.27)=.07x`    | `x*(.205-.16)=.045x`  | 
+| 12     | 16123.76        | 1128.67               | 725.57                |
```
Comment (xrchz) [inline assets/rpip-10/expense_2.md:14]: If we take the `.07x` seriously in the column heading, note that `07 × 16123.76 = 1128.6632 ≠ 1128.67`.

Also why are we rounding to 2 decimal places anyway?

File: assets/rpip-10/expense_2.md (hunk 1 above)
Comment (Valdorff) [reply assets/rpip-10/expense_2.md:16]: Added a small bit of explanation.

File: assets/rpip-10/expense_2.md (hunk 2 above)
Comment (Valdorff) [reply assets/rpip-10/expense_2.md:17]: Convenience and readability balanced against relevance. 0.005 RPL rounding error is negligible compared to the budgets in question.

Comment (xrchz) [reply assets/rpip-10/expense_2.md:17]: OK but we're off by 0.0068 RPL here, which is > 0.005.

Comment (Valdorff) [reply assets/rpip-10/expense_2.md:17]: Ah - got it. You're saying I rounded badly. Fair enough - fixing.

**Review by htimsk — APPROVED** [review]
Look accurate.

**Pull request merged**